### PR TITLE
Adding Italian flag cod

### PR DESCRIPTION
**Pull request by bernardopacini**
Adding recipe for Italian flag cod.

### CODE REVIEW

File: source/seafood/italianFlagCod.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* 1 sweet onion
+* Parsley
+* Salt and pepper, to taste
+* Olive oil
+
+Steps
+-----
+
+#. Preheat oven to 425.
+#. Place cod baking dish.
```
Comment (jennank) [inline source/seafood/italianFlagCod.rst]: Place cod in* baking dish?

File: source/seafood/italianFlagCod.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#. Chop tomatoes into 1/2 inch pieces.
+#. Chop onions into 1/4 inch pieces.
+#. Combine tomatoes and onions and cover cod in baking dish.
```
Comment (jennank) [inline source/seafood/italianFlagCod.rst]: I don't think you need baking dish twice, just above is good.

File: source/seafood/italianFlagCod.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+#. Sprinkle parsely, salt, and pepper over cod, tomatoes, and onion.
+#. Drizzle olive oil over cod, tomatoes, and onions.
```
Comment (jennank) [inline source/seafood/italianFlagCod.rst]: "Drizzle with olive oil" would remove some redundancy, but it's your call.

File: source/seafood/italianFlagCod.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+Steps
+-----
+
+#. Preheat oven to 425.
+#. Place cod baking dish.
+#. Chop tomatoes into 1/2 inch pieces.
+#. Chop onions into 1/4 inch pieces.
+#. Combine tomatoes and onions and cover cod in baking dish.
+#. Sprinkle parsely, salt, and pepper over cod, tomatoes, and onion.
```
Comment (jennank) [inline source/seafood/italianFlagCod.rst]: Typo: parsley*

**Review by jennank — CHANGES_REQUESTED** [review]

File: source/seafood/italianFlagCod.rst (hunk 2 above)
Comment (bernardopacini) [reply source/seafood/italianFlagCod.rst]: Agreed, updated.

File: source/seafood/italianFlagCod.rst (hunk 3 above)
Comment (bernardopacini) [reply source/seafood/italianFlagCod.rst]: Fixed.

File: source/seafood/italianFlagCod.rst (hunk 4 above)
Comment (bernardopacini) [reply source/seafood/italianFlagCod.rst]: Fixed.

File: source/seafood/italianFlagCod.rst [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+* Olive oil
+
+Steps
+-----
+
+#. Preheat oven to 425.
+#. Place cod in baking dish.
+#. Chop tomatoes into 1/2 inch pieces.
+#. Chop onions into 1/4 inch pieces.
+#. Combine tomatoes and onions and cover cod.
```
Comment (jennank) [inline source/seafood/italianFlagCod.rst]: I would put a comma instead of the second "and".

File: source/seafood/italianFlagCod.rst [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+#. Sprinkle parsley, salt, and pepper over cod, tomatoes, and onion.
```
Comment (jennank) [inline source/seafood/italianFlagCod.rst]: "Sprinkle with parsley, salt, and pepper" sounds better.

**Review by jennank — APPROVED** [review]

**Pull request merged**